fdbcli/FileConfigureCommand.actor.cpp

```diff
@@ -74,6 +71,12 @@ ACTOR Future<bool> fileConfigureCommandActor(Reference<IDatabase> db,
 		return false;
 	}
 
+	if (isNewDatabase) {
+		configString = "new" + configString;
+	} else {
+		configString.assign(configString.c_str() + 1);
```

Comment:
Can you use `configString.erase(0, 1);` here, as the intent is clearer and it will handle an empty string. 

Can you also add a comment that `configureStringFromJSON` adds a [leading space](https://github.com/apple/foundationdb/blob/901c09bfaab4c3a69ce3d730023076ede4536dc1/fdbclient/DatabaseConfiguration.cpp#L409) here and in function `configureStringFromJSON()`